example/example-buffers.py

```diff
@@ -30,3 +30,16 @@
     for j in range(m4.cols()):
         print(m4[i, j], end = ' ')
     print()
+
+from example import ConstructorStats
+cstats = ConstructorStats.get(Matrix)
```

Comment:
I don't think you can generally rely on things being immediately garbage collected when you assign ``None`` to a local. Invoking the garbage collector might be prudent.